creusot/tests/should_succeed/inc_max_3.rs

```diff
@@ -3,23 +3,30 @@
 
 extern crate creusot_contracts;
 use creusot_contracts::*;
-use std::mem::swap;
 
+#[requires(*ma <= 1000_000u32 && *mb <= 1000_000u32 && *mc <= 1000_000u32)]
+#[ensures(^ma != ^mb && ^mb != ^mc && ^mc != ^ma)]
 fn inc_max_3<'a>(ma: &'a mut u32, mb: &'a mut u32, mc: &'a mut u32) {
   if *ma < *mb {
-    swap(&mut ma, &mut mb);
```

Comment:
why did you inline this? can we not define a function `swap` locally that does what these 3 lines of code do?